specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -503,7 +482,7 @@ def verify_cell_kzg_proof_batch_impl(row_commitments: Sequence[KZGCommitment],
     #   RLP = sum_k (r^k * h_k^n) proofs[k]
     #
     # Here, the variables have the following meaning:
-    # - k < len(row_indices) is an index iterating over all cells in the input
+    # - k < len(cosets_evals) is an index iterating over all cells in the input
```

Comment:
Same as above, I think it's cleaner to use `len(cell_indices)` instead of `len(cosets_eval)` when we refer to iterating over all cells.